src/main.go

```diff
@@ -1055,6 +1131,16 @@ func main() {
 	flag.Parse()
 	tests := flag.Args()
 	startTime := time.Now()
+	if ll := os.Getenv("LOG_LEVEL"); ll != "" {
```

Comment:
Can't we use a flag like `-d`/`-debug` or something similar? I think that makes it easier to see from the usage output how to set the loglevel. We could use both if CI or something else depends on this.

Reply:
there is already a '-log-level' command line option/flag, does that work?